zilsd.adoc

```diff
@@ -39,9 +39,17 @@ Even if naturally aligned, the memory access might not be performed atomically.
 
 If the effective address is a multiple of 4, then each word access is required to be performed atomically.
 
-To ensure fault handling is possible for the load instructions, it must be ensured that the register which is the source of the base address is not overwritten before the entire operation is complete.
+The following table summarizes the required behavior:
+
+|===
+|Alignment |Word accesses guaranteed atomic? |Can cause misaligned trap?
```

Comment:
"Word" here means a 4 byte data unit, but is this clear enough for your average reader or should there be a footnote to clarify?

Reply:
If the average reader does not know what a word is, they will also not understand what a LW instructions is. So I think it is safe to assume they know.